exdb/templates/exdb/edit_experience.html

```diff
@@ -0,0 +1,40 @@
+{% extends "exdb/base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/edit_experience.css' %}" />
+{% endblock %}
+{% block content %}
+    {{ block.super }}
+    <h1>{% trans "Edit Experience" %}</h1>
+    <p><a href="{% url 'ra_home' %}">Back to Home</a></p>
+    {% if status == 'ad' %}
+        <p class="warning">Warning! Editing an approved experience will resubmit it for approval.</p>
+    {% endif %}
+    <span class="errors">{{ form.non_field_errors }}</span>
+    <span class="errors">{{ new_comment_form.non_field_errors }}</span>
```

Comment:
[`non_field_errors` is an iterable](https://docs.djangoproject.com/en/1.9/ref/forms/api/#django.forms.Form.non_field_errors). So you should probably be looping over it.
And since you're looping over both of them, you can use the [`add` template filter](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#add) to merge the list and loop over them at the same time.